docs/maven-executors.md

```diff
@@ -0,0 +1,3 @@
+# Maven Executors
```

Comment:
Since we don't have any info on Maven yet.
Lets remove it from there

Reply:
Same for Gradle

Reply:
Done.